composer.json

```diff
@@ -148,6 +148,7 @@
         "jackalope/jackalope": "< 1.3.4",
         "jackalope/jackalope-doctrine-dbal": "< 1.3.0",
         "jackalope/jackalope-jackrabbit": "< 1.3.0",
+        "jms/metadata": "2.5.2",
```

Comment:
version `2.5.2` causes a segfault at the moment: https://github.com/schmittjoh/metadata/pull/105#issuecomment-970271408